generate_new_lesions/augment_lesions_intra_dataset.py

```diff
@@ -0,0 +1,486 @@
+"""
+Use lesions from pathology subjects in dataset A and insert them into healthy controls within dataset A.
```

Comment:
how does it know which are patients and which are controls? maybe clarify in the function help